ngraph/core/src/op/equal.cpp

```diff
@@ -87,3 +87,9 @@ bool op::v1::Equal::evaluate(const HostTensorVector& outputs, const HostTensorVe
     NGRAPH_OP_SCOPE(v1_Equal_evaluate);
     return equal::evaluate_equal(inputs[0], inputs[1], outputs[0], get_autob());
 }
+
+bool op::v1::Equal::visit_attributes(AttributeVisitor& visitor)
+{
+    NGRAPH_OP_SCOPE(v1_Equal_visit_attributes);
```

Comment:
@ilyachur do we need this?

Reply:
Honestly in current case we don't need it because in the current case we don't get any binary size improvement. But I think we should keep it in order to have the one code style for all cases.